api/src/dataloaders/collections.rs

```diff
@@ -0,0 +1,43 @@
+use std::collections::HashMap;
+
+use async_graphql::{dataloader::Loader as DataLoader, FieldError, Result};
+use poem::async_trait;
+use sea_orm::prelude::*;
+
+use crate::{db::Connection, entities::collections, objects::Collection};
+
+#[derive(Debug, Clone)]
+pub struct ProjectLoader {
+    pub db: Connection,
+}
+
+impl ProjectLoader {
```

Comment:
should call ProjectCollectionsLoader and rename the file to project_collections.rs